test/Faker/Provider/da_DK/PaymentTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Faker\Test\Provider\da_DK;
+
+use Faker\Provider\da_DK\Payment;
+use PHPUnit\Framework\TestCase;
+use Faker\Generator;
+
+class PaymentTest extends TestCase
+{
+    private $faker;
+
+    public function setUp()
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Payment($faker));
+        $this->faker = $faker;
+    }
+
+    public function testBank()
+    {
+        $bankName = $this->faker->bank;
+        $this->assertNotEmpty($bankName);
+
+    }
+
+    public function testLocalBankAccountNumber()
+    {
+        $accountNo = $this->faker->localBankAccountNumber;
+        $this->assertEquals(10, strlen($accountNo));
```

Comment:
This is a weak assertion, I'd prefer if you could validate the number against a regular expression coming from a public format documentation